monitoring_service/test/test_e2e.py

```diff
@@ -106,29 +106,29 @@ def test_e2e(
     )
 
     monitoring_service.transport.receive_fake_data(monitor_request.serialize_full())
-    gevent.sleep(0)
+    gevent.sleep(0.01)
     assert channel_id in monitoring_service.monitor_requests
 
     c2.close_channel(c1.address, balance_proof_c1)
-    # wait till monitor's blockchain confirms closeChannel event...
-    wait_for_blocks(10)
-    gevent.sleep(1)
-    # wait till validator confirms NewBalanceProofReceived event...
-    wait_for_blocks(10)
-    gevent.sleep(1)
+    # Wait one block until the ChannelClosed event is confirmed and handled
+    # by the MS
+    wait_for_blocks(1)
+    # Now give the monitoring service a chance to submit the missing BP
+    gevent.sleep(0.01)
```

Comment:
Do we need a wait here or just a context switch (like `sleep(0)`)?

Reply:
We might have to wait for the `poll_interval` to elapse. Otherwise the MS might still be inside a `sleep(poll_interval)` and not wake up on the next context switch. Explicitly using a `TEST_POLL_INTERVAL` here would probably make that more clear.